oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/BanyanDB.java

```diff
@@ -91,4 +91,19 @@
     @interface NoIndexing {
 
     }
+
+    /**
+     * Additional information for constructing Index in BanyanDB.
+     *
+     * @since 9.3.0
+     */
+    @Target({ElementType.FIELD})
+    @Retention(RetentionPolicy.RUNTIME)
+    @interface IndexRule {
+        IndexType indexType() default IndexType.INVERTED;
+
+        enum IndexType {
+            INVERTED, TREE;
```

Comment:
Please provide a simple explanation for those two index types.

Reply:
Added